RoaringBitmap/src/test/java/org/roaringbitmap/TestSerialization.java

```diff
@@ -208,39 +208,20 @@ public void testImmutableBuildingBySerialization() {
     presoutbb.position(presoutbb.position() + imrempty.serializedSizeInBytes());
     assertEquals(imrempty.isEmpty(), true);
     ImmutableRoaringBitmap imrb = new ImmutableRoaringBitmap(presoutbb);
-    int cksum1 = 0, cksum2 = 0, count1 = 0, count2 = 0;
-    for (int x : bitmap_a) { // or bitmap_a1 for a version without run
-      cksum1 += x;
-      ++count1;
-    }
-    for (int x : imrb) {
-      cksum2 += x;
-      ++count2;
-    }
+    int cksum1[] = {0}, cksum2[] = {0}, count1[] = {0}, count2[] = {0};
```

Comment:
Given we are in a test, I would have gone for a double iteration, andf go through:

`bitmap_a.stream().sum()`; (by the way, here, we do not care with int overflow).
`bitmap_a.stream().count()`.